lib/howl/ui/process_buffer.moon

```diff
@@ -54,7 +54,7 @@ ProcessMode = {
           else
             process\send_signal 'INT'
             log.info "Sent SIGINT to #{process.pid}"
-       else
+        else
```

Comment:
Out of curiosity, what do this and the `flair.moon` changes have to do with Moonscript 0.5?